benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -1593,9 +1607,9 @@ void GraphicsBenchmarkApp::RecordCommandBufferSpheres(PerFrame& frame)
     frame.cmd->BindGraphicsDescriptorSets(mSphere.pipelineInterface, 1, &mSphere.descriptorSets.at(GetInFlightFrameIndex()));
 
     // Snapshot some scene-related values for the current frame
-    uint32_t currentSphereCount   = pSphereInstanceCount->GetValue();
+    uint32_t currentSphereCount   = std::min<uint32_t>(pSphereInstanceCount->GetValue(), mInitializedSpheres);
```

Comment:
in what case `pSphereInstanceCount` could be larger than `mInitializedSpheres`?

Reply:
It can occur when we drag the `sphereCount` slider, because we only update the `initializedSpheres` value once the slider is released. In other words, we can drag the slider, and `sphereCount` can increase, but `initializedSpheres` would remain the same up to the slider is let go.